pkgs/applications/science/misc/root/default.nix

```diff
@@ -38,15 +38,16 @@
 , CoreSymbolication
 , OpenGL
 , noSplash ? false
+, swVersPatch ? ./sw_vers.patch
```

Comment:
This is a bit unusual

Reply:
The motivation is to make version (source) overriding easier.

I have a flake.nix in my analysis project that tracks the master branch of root-project/root. The ./sw_vers.patch is a moving target that needs to be regenerated from time to time.

I acknowledge that it's kinda "unusual". I would change it back if need be.

Reply:
I also often maintain custom overrides. In my experience with packaging bleeding edge version, I often just end up dropping all values from the original `patches` attribute.

The problem with doing one-off interfaces is that there will be other packages you might want to override and your existing experience around `swVersPatch` will not translate to that.

Reply:
I see.